src/edu/csus/ecs/pc2/core/execute/IExecuteFrameNotify.java

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 1989-2019 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.core.execute;
+
+/**
+ * @author John Buck
+ *
+ */
+public interface IExecuteFrameNotify {
+
+    // Terminate button pressed
```

Comment:
change // to java doc comment, perhaps comment like:
Notify Observer that auto judging has been terminated.

Reply:
code still contains // Terminate button pressed
should be java doc.

Reply:
The github diff used, shows the // comment was still there.

Reply:
confirmed fix, reviewed code directly on branch and it is fixed.